### PR TITLE
[glm] Fixed missing symbols and crosshair

### LINKED ISSUE #236: [android] [bug] Сrosshair is not displayed after "Add place" (10.6.0-Web-beta (51060))
When the user clicks "add place", the crosshair is not displayed (to select the location point).
At the same time, you can still add the place itself. The problem is that you can't see exactly where to place it.

Android, 10.6.0-Web-beta (51060)

### LINKED ISSUE #237: Missing numbers on the map scale
image

CC: @vng

### PR DESCRIPTION
**Pull request by biodranik**
Drape implementation doesn't like this commit (which disables default initialization): https://github.com/g-truc/glm/commit/4cf8a10af2bba678c2ad136006e6ec41118b6746#diff-c5606b5d108d3562fa085b09df701f4c6472a58106d8414b0b2beaafbe82eeebL472

Fixes #236
Fixes #237
Fixes #243

### CODE REVIEW

File: drape_frontend/drape_engine.cpp [hunk 1]
```diff
@@ -55,9 +55,9 @@ DrapeEngine::DrapeEngine(Params && params)
 
   std::vector<PostprocessRenderer::Effect> effects;
 
-  bool enabledAntialiasing;
-  if (!settings::Get(dp::kSupportedAntialiasing, enabledAntialiasing))
-    enabledAntialiasing = false;
+//  bool enabledAntialiasing;
+//  if (!settings::Get(dp::kSupportedAntialiasing, enabledAntialiasing))
+//    enabledAntialiasing = false;
```
Comment (vng) [inline drape_frontend/drape_engine.cpp:61]: So according to the code below, we always work without AA? effects vector is empty ..

**Comment by biodranik** [comment]
Yep

**Comment by pastk** [comment]
JFYI, this fix works for me! The scale number, the crosshair and the north facing arrow are back!

File: drape_frontend/drape_engine.cpp (hunk 1 above)
Comment (biodranik) [reply drape_frontend/drape_engine.cpp:61]: Yes

**Review by vng — APPROVED** [review]

**Pull request merged**